css/css-text/writing-system/writing-system-font-001.html

```diff
@@ -0,0 +1,25 @@
+<!doctype html>
+<html lang=en>
+<meta charset=utf-8>
+<title>CSS test: font selection and writing sytem</title>
+<link rel=author title="Ambrose Li" href="http://port.ambroseli.ca/">
+<link rel=author title="Florian Rivoal" href="https://florian.rivoal.net">
+<link rel=help href="https://drafts.csswg.org/css-text-3/#languages">
+<link rel=help href="https://drafts.csswg.org/css-text-3/#script-tagging">
+<link rel=help href="https://drafts.csswg.org/css-fonts-4/#language-specific-support">
+<link rel=match href="reference/writing-system-font-001-ref.html">
+<meta name=assert content="An explicit writing system takes precedence over the content language for font selection">
```

Comment:
This comment doesn't seem to match the test, which asserts (effectively) that `lang=foo` has no effect at all in this case.

Reply:
The test does not assert that `lang=foo` has no effect, but that if you have `lang=foo-bar` where `foo` is a language and `bar` is a writing system, it's `bar` that wins, not `foo`.

So unless I'm missing something, the comment is correct.

Reply:
Oh, so the `-latn` bit is the important part. I've just revealed my ignorance about the area, but if nobody else will review, I will :)